lib/dash/dash_parser.js

```diff
@@ -486,8 +486,13 @@ shaka.dash.DashParser.prototype.processManifest_ =
   let baseUris = shaka.util.ManifestParserUtils.resolveUris(
       manifestBaseUris, uris);
 
-  let minBufferTime =
+  let ignoreMinBufferTime = this.config_.dash.ignoreMinBufferTime;
+  let minBufferTime = 0;
+  if (!ignoreMinBufferTime) {
+    minBufferTime =
       XmlUtils.parseAttr(mpd, 'minBufferTime', XmlUtils.parseDuration);
```

Comment:
`parseAttr()` can return `null` if there's no such attribute, so please add `|| 0` to the end of this statement.